packages/rps/src/redux/message-service/saga.ts

```diff
@@ -32,29 +36,63 @@ export default function* messageSaga() {
   yield fork(exitGameSaga);
 }
 
-interface Message {
+interface CommitmentMessage {
   data: Commitment;
   signature: string;
   queue: Queue;
   userName?: string;
 }
 
+interface MessageMessage {
```

Comment:
Not a big fan of this name. "NonProtocolMessage"? "NonCommitmentMessage"? "MessageForWallet"?

Reply:
Also not a fan of this name, which is a motivation for this review. The problem is that this is a message to YourApp containing a message to YourWallet. `NonCommitmentMessage` seems the best out of your suggestions, and I don't have a better one.

Reply:
Let's go with `NonCommitmentMessage` for now?